Tests/RevenueCatUITests/CustomerCenter/ManageSubscriptionsViewModelTests.swift

```diff
@@ -39,9 +40,15 @@ class ManageSubscriptionsViewModelTests: TestCase {
         }
     }
 
+    override func setUp() {
+        super.setUp()
+        testClock = TestClock()
+    }
```

Comment:
Perhaps this is not needed since you're already initializing `testClock` in the variable declaration